packs/elasticsearch/config.yml

```diff
@@ -0,0 +1,18 @@
+name: Elasticsearch
+
+description: |+
+  This template pack includes dashboards and alerts for popular signals regarding Elasticsearch cluster health and performance.
```

Comment:
Small suggestion: 'This observability pack includes...'

Reply:
nice catch!